jme3-core/src/main/resources/Common/MatDefs/Misc/Unshaded.frag

```diff
@@ -45,6 +47,11 @@ void main(){
            discard;
         }
     #endif
+    
+    #ifdef DESATURATE

```

Comment:
I am not a shader guy here, but shouldn't this be `#ifdef DESATURATION`?

Reply:
Yes you're right, thanks for the catch, I updated the name of the define so the spelling is consistent